tests/system/providers/google/cloud/vertex_ai/example_vertex_ai_auto_ml.py

```diff
@@ -0,0 +1,688 @@
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# mypy ignore arg types (for templated fields)
+# type: ignore[arg-type]
+
+"""
+Example Airflow DAG for Google Vertex AI service testing Auto ML operations.
+"""
+import os
+from datetime import datetime
+from pathlib import Path
+
+from google.cloud.aiplatform import schema
+from google.protobuf.json_format import ParseDict
+from google.protobuf.struct_pb2 import Value
+
+from airflow import models
+from airflow.operators.bash import BashOperator
+from airflow.providers.google.cloud.operators.gcs import GCSCreateBucketOperator, GCSDeleteBucketOperator
+from airflow.providers.google.cloud.operators.vertex_ai.auto_ml import (
+    CreateAutoMLForecastingTrainingJobOperator,
+    CreateAutoMLImageTrainingJobOperator,
+    CreateAutoMLTabularTrainingJobOperator,
+    CreateAutoMLTextTrainingJobOperator,
+    CreateAutoMLVideoTrainingJobOperator,
+    DeleteAutoMLTrainingJobOperator,
+    ListAutoMLTrainingJobOperator,
+)
+from airflow.providers.google.cloud.operators.vertex_ai.dataset import (
+    CreateDatasetOperator,
+    DeleteDatasetOperator,
+    ImportDataOperator,
+)
+from airflow.providers.google.cloud.transfers.local_to_gcs import LocalFilesystemToGCSOperator
+from airflow.utils.trigger_rule import TriggerRule
+
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+DAG_ID = "vertex_ai_auto_ml_operations"
+PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT", "default")
+REGION = "us-central1"
+
+FORECAST_GCS_BUCKET_NAME = f"bucket_forecast_{DAG_ID}_{ENV_ID}"
+IMAGE_GCS_BUCKET_NAME = f"bucket_image_{DAG_ID}_{ENV_ID}"
+TABULAR_GCS_BUCKET_NAME = f"bucket_tabular_{DAG_ID}_{ENV_ID}"
+TEXT_GCS_BUCKET_NAME = f"bucket_text_{DAG_ID}_{ENV_ID}"
+VIDEO_GCS_BUCKET_NAME = f"bucket_custom_python_{DAG_ID}_{ENV_ID}"
+
+FORECAST_ZIP_CSV_FILE_LOCAL_PATH = str(Path(__file__).parent / "resources" / "forecast-dataset.csv.zip")
+IMAGE_ZIP_CSV_FILE_LOCAL_PATH = str(Path(__file__).parent / "resources" / "image-dataset.csv.zip")
+TABULAR_ZIP_CSV_FILE_LOCAL_PATH = str(Path(__file__).parent / "resources" / "tabular-dataset.csv.zip")
+TEXT_ZIP_CSV_FILE_LOCAL_PATH = str(Path(__file__).parent / "resources" / "text-dataset.csv.zip")
+VIDEO_ZIP_CSV_FILE_LOCAL_PATH = str(Path(__file__).parent / "resources" / "video-dataset.csv.zip")
```

Comment:
I know it's not your change (@VladaZakharova ) but could you use one common variable for resources path:
```suggestion
RESOURCES_PATH = Path(__file__).parent / "resources"
FORECAST_ZIP_CSV_FILE_LOCAL_PATH = str(RESOURCES_PATH / "forecast-dataset.csv.zip")
IMAGE_ZIP_CSV_FILE_LOCAL_PATH = str(RESOURCES_PATH / "image-dataset.csv.zip")
TABULAR_ZIP_CSV_FILE_LOCAL_PATH = str(RESOURCES_PATH / "tabular-dataset.csv.zip")
TEXT_ZIP_CSV_FILE_LOCAL_PATH = str(RESOURCES_PATH / "text-dataset.csv.zip")
VIDEO_ZIP_CSV_FILE_LOCAL_PATH = str(RESOURCES_PATH / "video-dataset.csv.zip")
```

Reply:
It is really good idea, thanks!